config/configuration.toml.in

```diff
@@ -10,6 +10,10 @@ machine_type = "@MACHINETYPE@"
 # trouble running pre-2.15 glibc
 kernel_params = "@KERNELPARAMS@"
 
+# Path to the BIOS.
+# If you want that qemu uses the default BIOS leave this option empty
```

Comment:
Do we need to say anything in this comment about `machine_type`? Will this only work for `q35`? Will it work for any value? *etc*.

Reply:
This looks to me to be independant of machine type and q35 - so I'd drop the mention of q35 or machine type at all from the series - just say it allows a choice of BIOS.